src/main/resources/quests.yml

```diff
@@ -76,4 +76,25 @@ quests:
             - "rod"
           fish-to-catch: 5
     rewards:
-      exp: 250
\ No newline at end of file
+      exp: 250
+
+  TimedQuest:
+    name: 'Timed Quest'
+    ask-message: 'This is timed quest, mine dirt before time runs out!'
+    finish-message: 'You did it in time!'
+    stages:
+      ordered:
+        '1':
+          break-block-names:
+          - DIRT
+          break-block-amounts:
+          - 10
+          break-block-durability:
+          - 0
+          start-event: TimerEvent
+          finish-event: CancelTimer
+        '2':
+          password-displays:
+          - "Do you want to finish? (Type yes before the time runs out!)"
```

Comment:
Normally user doesn't need to confirm finish. It is not required for timers to work correctly. I originally added the second stage to test out if the CancelTimer event fires and works properly.